reference/plugins/available_plugins.rst

```diff
@@ -691,11 +691,23 @@ Administration API
 Bulk Data
```

Comment:
I think the $document operation needs to be listed on this page as well.

Reply:
Done!